ruma-federation-api/src/backfill/get_backfill/v1.rs

```diff
@@ -0,0 +1,45 @@
+//! [GET /_matrix/federation/v1/backfill/{roomId}](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-backfill-roomid)
+
+use std::time::SystemTime;
+
+use js_int::UInt;
+use ruma_api::ruma_api;
+use ruma_events::pdu::Pdu;
+use ruma_identifiers::{EventId, RoomId};
+
+ruma_api! {
+    metadata: {
+        description: "Request more history from another homeserver",
+        name: "get_backfill",
+        method: GET,
+        path: "/_matrix/federation/v1/backfill/:roomId",
+        rate_limited: false,
+        requires_authentication: true,
+    }
+
+    request: {
+        /// The room ID to backfill.
+        #[ruma_api(path)]
+        pub room_id: RoomId,
+
+        /// The event IDs to backfill from.
+        #[ruma_api(query)]
+        pub v: Vec<EventId>,
```

Comment:
This isn't documented well yet (I really need to get to that with the amount of endpoint PRs we're getting right now!), but could you update this to the new API conventions of borrowing in the request block and having constructors for the generated `Request` and `Response` (as in https://github.com/ruma/ruma/commit/dffa60d70f6d76306c117e29dab1e73cb8f5acd6)? Specifically for the former (since it's very much non-obvious):

```suggestion
        /// The room ID to backfill.
        #[ruma_api(path)]
        pub room_id: &'a RoomId,

        /// The event IDs to backfill from.
        #[ruma_api(query)]
        pub v: &'a [EventId],
```